backend/tests/e2e/test_parallel_multi_agent_e2e.py

```
@@ -0,0 +1,589 @@
+                f"{message_label} response did not complete within {max_wait}s. "
+                f"Started with {num_stages_before} stages, now have {len(chat_stages)} total, "
+                f"{len(new_stages)} new (unverified) stages: {debug_info}"
+            )
+        
+        return chat_stage
+
+    async def _verify_chat_response(
+        self,
+        chat_stage,
+        message_key: str,
+        expected_conversation: dict
```

Ruff (0.14.8)

Unused method argument: `expected_conversation`

(ARG002)